lib/IRGen/GenProto.cpp

```diff
@@ -4651,7 +4655,7 @@ llvm::Constant *IRGenModule::getAddrOfGenericEnvironment(
         fields.addAlignmentPadding(Alignment(4));
 
         // Generic requirements
-        irgen::addGenericRequirements(*this, fields, signature);
+        irgen::addGenericRequirements(*this, fields, signature, reqs, inverses);
```

Comment:
Do the other callers of the three-argument overload need to be audited to see if they need the same fix? Maybe it should be removed?